src/server/redis_connection.cc

```diff
@@ -163,6 +163,15 @@ std::string Connection::MultiBulkString(const std::vector<std::string> &values,
   return result;
 }
 
+std::string Connection::ArrayOfSet(const std::vector<std::string> &elems) const {
```

Comment:
IMHO I cannot understand the name.

It is just a **set**, not an **array of sets**. Instead, it is a **set of bulk strings**.

An **array of sets** is typed `Array<Set<T>>`.

Reply:
@PragmaTwice So do you suggest changing this name to SetOfBulkStrings? If yes, I will bring this fix in the next RESP3 PR.

Reply:
Yeah that's better to me. ArrayOfSet seems like an array composed by sets.

Reply:
Thanks for your good suggestion.